packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -331,14 +332,28 @@ class InspectorPreferencesController extends DisposableController
     }
   }
 
-  void _cacheCustomPubRootDirectories(
+  void _cachePubRootDirectories(
```

Comment:
return `Future<void>` now that we are awaiting in this method so that we can catch unawaited futures